samples/client/petstore/java/jersey2-java6/src/main/java/io/swagger/client/ApiClient.java

```diff
@@ -432,13 +433,12 @@ public String parameterToString(Object param) {
    *   application/json; charset=UTF8
    *   APPLICATION/JSON
    *   application/vnd.company+json
-   * "* / *" is also default to JSON
    * @param mime MIME
    * @return True if the MIME type is JSON
    */
   public boolean isJsonMime(String mime) {
     String jsonMime = "(?i)^(application/json|[^;/ \t]+/[^;/ \t]+[+]json)[ \t]*(;.*)?$";
-    return mime != null && (mime.matches(jsonMime) || mime.equals("*/*"));
+    return mime != null && (mime.matches(jsonMime) || mime.equalsIgnoreCase("application/json-patch+json"));
```

Comment:
Rebased but same diff. I think you didn't change `ApiClient.mustache` under `Java` with this commit https://github.com/swagger-api/swagger-codegen/commit/927055a681fb405eb202577cd6b0c6805662e3b8. Only under `Java/libraries` folder